xml2rfc/writers/pdf.py

```diff
@@ -123,13 +123,14 @@ def write(self, filename):
     def get_serif_fonts(self):
         fonts = set()
         scripts = self.root.get('scripts').split(',')
-        noto_serif = "Noto Serif"
+        noto_serif = 'Noto Serif'
+        noto_symbols = 'NotoSansSymbols2'
         for script in scripts:
             family = get_noto_serif_family_for_script(script)
-            fonts.add("%s" % family)
+            fonts.add('%s' % family)
```

Comment:
why the shift from " to ' in this PR? - we've got some push to move to black's standards, which want ".

Reply:
I felt wrong to keep that method with mixed quotes.
I can get rid of these.

Reply:
TIL balk prefers double quotes over single. :(
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings

Reply:
Left quotes as it is and used "" quotes for updates.